OracleWebCenterContent/dockerfiles/README.MD

```diff
@@ -0,0 +1,315 @@
+
+This repository includes quick start dockerfiles for WebCenter Content 12.2.1.4.0 based on Oracle Linux 7, Oracle JRE 8 (Server) and Oracle WebLogic Infrastructure 12.2.1.4.0.
+
+The containers will be connected using a Docker User Defined network.
+More information on Docker and its installation in Oracle Linux can be found here:
+https://community.oracle.com/docs/DOC-914969
+
+# 2. Hardware and Software Requirements
+Oracle WebCenter Portal has been tested and is known to run on the following hardware and software:
+
+## 2.1. Hardware Requirements
+
+| Hardware  | Size  |
+| :-------: | :---: |
+| RAM       | 16GB  |
+| Disk Space| 200GB+|
+
+## 2.2. Software Requirements
+
+|       | Version                        | Command to verify version |
+| :---: | :----------------------------: | :-----------------------: |
+| OS    | Oracle Linux 7.3 or higher     | more /etc/oracle-release  |
+| Docker| Docker version 17.03 or higher | docker version           |
+
+# 3. Pre-requisites
+
+## 3.1. To confirm that userid is part of the Docker group, run the below command:
+```
+   $ sudo id -Gn <userid>
+```
+* Run 'docker ps -a' command to confirm user is able to connect to Docker engine.
+To add/modify user to be part of docker group
+```
+   $ sudo /sbin/usermod -a -G docker <userid>
+```
+
+## 3.2. Set the Proxy if required:
+
+Set up the proxy for docker to connect to the outside world - to access external registries and build a Docker image, set up environment variables for proxy server something like this:
+```
+   export http_proxy=http://www-yourcompany.com:80 
+   export https_proxy=http://www-yourcompany.com:80 
+   export HTTP_PROXY=http://www-yourcompany.com:80 
+   export HTTPS_PROXY=http://www-yourcompany.com:80 
+   export NO_PROXY=localhost,.yourcompany.com 
+```
+
+## 3.3. Create a User Defined network
+
+In this configuration creation of a user defined network will enable communication between the containers just using container names. User defined network option was preferred over the container linking option as the latter is now deprecated. For this setup we will use a user defined network using bridge driver.Create a user defined network using the bridge driver by executing the following command:
+
+        $ docker network create -d bridge <some name>
+
+Sample command ...
+
+        $ docker network create -d WCContentNetwork
+
+## 3.4. Mount a host directory as a data volume
+You need to mount volumes, which are directories stored outside a container's file system, to store WebLogic domain files and any other configuration files. The default location of the volume in the container is `/var/lib/docker/volumes`. 
+
+This option lets you mount a directory from your host to a container as volume. This volume is used to mainly store WebLogic server domain files. The volume is created at this location `/scratch/wccdocker/volumes/wccontentvolume`.
```

Comment:
Is this inside the container or on the host file system? It's not clear to me. If it's meant to be on the host, please don't refer to `/scratch` as that's not used very often. Just mention that any host directory can be bind mounted into in the container using `-v`